### DISCUSSION TITLE
禁用评论区视频专栏链接转换

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1595] [功能建议 / Ideas]
https://github.com/the1812/Bilibili-Evolved/issues/555#issuecomment-730271044
说上面这句话的时候还是半开玩笑性质的。现在还是想认真探讨一下有没有这个必要。

我个人的使用体验感受是五五开。
提这个需求主要还是这玩意 B 站做得一般般，一部分需求是 [之前提过的](https://github.com/the1812/Bilibili-Evolved/issues/555#issuecomment-730273080)。
在这完整地展开说说。
禁用转换方便地一眼看出：
- 原文本到底是 av 号还是 BV 号还是别的啥的乱七八糟的链接 ~（废话）~
- 跳转至的分 p 
- 跳转时间
- 指向评论（至少知道发送者意思是指向评论，如果发送者知道有评论链接的话）
- 具体弹幕（不过这个好像只能跳转自己的弹幕🌚）
- 番剧影视链接是 ss 号（某部番剧/影视）还是 ep 号（具体某一集）还是番剧影视详情页（md）（同上，如果发送者知道区别的话）
- 还有一些奇奇怪怪的和旧版域名相关的情景，如
![image](https://user-images.githubusercontent.com/49396421/108856667-3fbab400-7625-11eb-986d-06e9964c06f0.png)

**Comment by Geeyun-JY3** [comment]
~~还能获得最原汁原味的被 Rickroll  体验~~

**Reply by the1812** [reply]
专业 Rickroll 当然标题也会改的(
